woocommerce/woocommerce-paypal-payments
# Update the apple-developer-merchantid-domain-association validation strings for Apple Pay (3158)

### Description

This PR updates expired ApplePay validation strings.

### Steps to Test

1. Configure the Apple Pay Payments in Sandbox and Live modes and ensure you can register the domain correctly.


### Screenshots
N/A
